draft-ietf-tls-tls13.md

```diff
@@ -5045,17 +5051,20 @@ in \[].
                                v
                             NEGOTIATED
                                | Send ServerHello
+                               | K_send = handshake
                                | Send EncryptedExtensions
                                | [Send CertificateRequest]
 Can send                       | [Send Certificate + CertificateVerify]
 app data -->                   | Send Finished
-after                 +--------+--------+
-here         No 0-RTT |                 | 0-RTT
-                      |                 v
-                      |             WAIT_EOED <---+
+after                          | K_send = application
+here                  +--------+--------+
```

Comment:
Now that you have K_send = application, you could just say that after installing application keys in K_send, then you can send application data.  Same goes for receiving it.

At a minimum, I would move the arrow to the line where you install the keys.